src/ui/CardTabTitleOtions/CardTabTitleOtions.jsx

```diff
@@ -0,0 +1,133 @@
+import { Options } from "../Options/Options";
+import "./CardTabTitleOptions.css";
+import { useEffect, useState } from "react";
+import axios from "axios";
+
+const BASE_URL = "https://bootcamp-api.codeit.kr/api";
+
+export function CardTabTitleOptions({ links }) {
+  const [tabList, setTabList] = useState([]);
+  const [clickTabId, setClickTabId] = useState(0);
+  const [clickTabAll, setClickTabAll] = useState(true);
+  const [tabName, setTabName] = useState("전체");
+
+  // tab목록 가져오기
+  async function fetchTabList() {
+    try {
+      const response = await axios.get(`${BASE_URL}/users/1/folders`);
+      const { data } = response;
+      setTabList(data.data);
+    } catch (error) {
+      throw new Error(error);
+    }
+  }
+
+  useEffect(() => {
+    fetchTabList();
+  }, []);
+
+  // 탭들을 클릭시 작동 핸들러
+  const handleSelectTab = (e) => {
+    setClickTabAll(false);
+    setClickTabId(e.target.id);
+
+    const getTab = tabList.filter((tab) => tab.id == e.target.id);
```

Comment:
- getTab은 함수 이름 같습니다. 탭 정보를 담고 있는 변수이니 `selectedTab` 정도의 이름이 더 적절해 보입니다.  
- 조건에 맞는 탭 하나만을 찾는 코드이니 filter 보다는 find 함수를 사용하는게 좋겠습니다.    
- `==` 연산자는 비교를 위해 임의의 형변환이 일어나기 때문에 의도하지 않은 코드 실행 결과가 나올 확률이 높습니다. 두 변수를 비교할 때는 주로 `===` 연산자를 사용합니다. 
```suggestion
    const selectedTab = tabList.find((tab) => tab.id === e.target.id);
```

Reply:
find함수를 오랫동안 사용하지 않아서 잊고 있었습니다! 감사합니다!